.stylelintrc

```diff
@@ -33,6 +33,6 @@
       "except": ["first-nested"],
       "ignore": ["after-comment"]
     } ],
-    "unit-whitelist": ["em", "rem", "%", "s", "deg", "ms", "vh"]
+    "unit-allowed-list": ["em", "rem", "%", "s", "deg", "ms", "vh", "px"]
```

Comment:
we took 'px' off the whitelist originally to force devs to use 'rem' or 'em' - theres' a mixin for px to rem conversion eg: @include rem(margin, 15);

Reply:
Ok that's great, I agree with that, reversed the change.